tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -2011,6 +2011,65 @@ public function userCopiesOrMovesFileWithFileIdFromAndToSpaceBetweenSpaces(strin
 		}
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has (copied|moved|renamed) a file "([^"]*)" into "([^"]*)" inside space "([^"]*)" using file-id path "([^"]*)"$/
+	 *
+	 * @param string $user
+	 * @param string $actionType
+	 * @param string $sourceFile
+	 * @param string $destinationFile
+	 * @param string $toSpaceName
+	 * @param string $url
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userHasRenamedAFileIntoUsingFileIdPath(string $user, string $actionType, string $sourceFile, string $destinationFile, string $toSpaceName, string $url): void {
+		$sourceFile = \trim($sourceFile, "/");
+		$sourceFile = \explode("/", $sourceFile);
+		$sourceFile = \end($sourceFile);
+		$destinationFile = \trim($destinationFile, "/");
+		$fileDestination = '';
+		if ($actionType === 'copied' || $actionType === 'moved') {
+			$fileDestination = $this->escapePath($destinationFile) . '/' . $this->escapePath($sourceFile);
+		} else {
+			$fileDestination = $destinationFile;
+		}
+		$baseUrl = $this->featureContext->getBaseUrl();
+		if ($toSpaceName === 'Shares') {
+			$sharesPath = $this->featureContext->getSharesMountPath($user, $fileDestination);
+			$davPath = WebDavHelper::getDavPath($user, $this->featureContext->getDavPathVersion());
+			$headers['Destination'] = $baseUrl . "/$davPath" . $sharesPath;
+		} else {
+			$headers['Destination'] = $this->destinationHeaderValueWithSpaceName($user, $fileDestination, $toSpaceName, $url);
+		}
+		$fullUrl = $baseUrl . $url;
+		if ($actionType === 'copied') {
+			$response = $this->copyFilesAndFoldersRequest($user, $fullUrl, $headers);
+		} else {
+			$response = $this->moveFilesAndFoldersRequest($user, $fullUrl, $headers);
+		}
+		Assert::assertEquals(
+			201,
+			$response->getStatusCode(),
+			"Response did not return expected status code"
+		);
+	}
+
+	/**
+	 * @Given user :user has sent HTTP method :verb to URL :url with content :content
+	 *
+	 * @param string $user
+	 * @param string $url
+	 * @param string $content
+	 *
+	 * @return void
+	 */
+	public function userHasEditedAFileWithContentInsideSpaceUsingFileIdPath(string $user, string $url, string $content): void {
+		$response = $this->featureContext->sendingToWithDirectUrl($user, 'PUT', $url, $content);
```

Comment:
```suggestion
		$response = $this->featureContext->sendingToWithDirectUrl($user, $verb, $url, $content);
```
`verb` is unused in the step def. The scenario step has `PUT` method but it is hardcoded in the code. Use the variable `verb`.